router/router.go

```diff
@@ -0,0 +1,24 @@
+package router
+
+import (
+	"os"
+
+	"github.com/gin-gonic/gin"
+)
+
+func Initialize() {
+	// Initialize router
```

Comment:
Remove comment.

Reply:
Why do you think removing this comment is a good alternative?

Instead of removing, because people who is starting learning golang can understand better what `gin.Default()` is, I added a better comment to the whole file